tests/Symfony/RequiredAutowiringExtensionTest.php

```diff
@@ -0,0 +1,71 @@
+<?php declare(strict_types = 1);
+
+namespace PHPStan\Symfony;
+
+use PHPStan\Reflection\AdditionalConstructorsExtension;
+use PHPStan\Reflection\ConstructorsHelper;
+use PHPStan\Rules\Properties\UninitializedPropertyRule;
+use PHPStan\Rules\Rule;
+use PHPStan\Testing\RuleTestCase;
+use Symfony\Contracts\Service\Attribute\Required;
+use function class_exists;
+
+/**
+ * @extends RuleTestCase<UninitializedPropertyRule>
+ */
+final class RequiredAutowiringExtensionTest extends RuleTestCase
+{
+
+	protected function getRule(): Rule
+	{
+		$container = self::getContainer();
+		$container->getServicesByTag(AdditionalConstructorsExtension::EXTENSION_TAG);
+
+		return new UninitializedPropertyRule(
+			new ConstructorsHelper(
```

Comment:
Please fetch the rule with `$container->getByType(UninitializedPropertyRule::class)`. It should be possible. That way it doesn't have to be in the baseline.